lib/dpla/map/factories.rb

```diff
@@ -10,7 +10,7 @@
     date { |timespan| timespan.association :timespan, :strategy => :build }
     description 'Window of the Stonewall Bar N.Y. 1969. The other half of the graffiti was erased by the time Diana photographed it.'
     extent '10x12 cm'
-    format 'Silver Gelatin Print'
+    dcformat 'Silver Gelatin Print'
```

Comment:
I'm circling back around to this finally. This failed when I was prototyping the [BHL mapping](https://gist.github.com/anarchivist/d46790e1029df7440708#file-bhl_mapping-rb-L36-L38) with `TypeError: no implicit conversion of Proc into String`.

I don't know if a test is appropriate here (i.e., in `dpla_map`), or if it makes more sense in Krikri. It seems like the issue is making sure that the DSL method doesn't conflict with a method name that exists somewhere else, like [`Kernel#format`](http://www.ruby-doc.org/core-2.1.5/Kernel.html#method-i-format). I'd lean towards the burden being on Krikri rather than `dpla_map`.